src/components/AppGrid.vue

```diff
@@ -49,7 +49,9 @@
           :key="child"
           :class="'child' + i"
           :style="{ gridArea: child }"
-        ></div>
+        >
+          <button @click="removeChild(i)">(x)</button>
```

Comment:
just an `x` is enough I would say.. no need for `()`

Reply:
what about `&times;` ?